src/compas_model/model/interactiongraph.py

```diff
@@ -84,7 +91,8 @@ def edge_interaction(self, edge):
         """
         return self.edge_attribute(edge, "interaction")  # type: ignore
 
-    def print_interactions(self):
+    def interactions(self):
```

Comment:
i dont' think this makes sense.`InteractionGraph.interactions` should not return a string, but rather a list of interactions.

Reply:
agree that's weird, but this previously did the same just using `print`. I renamed this to `interactions_str` as it really just builds a string form of the interactions. 

What would you have `InteractionGraph.interactions` return? just a flat list of `Interaction`s?

Reply:
yes, a flat list of all interaction objects for easy access. we do the same, for example, with elements in the model...